src/System.DirectoryServices.Protocols/tests/DirectoryAttributeModificationCollectionTests.cs

```diff
@@ -0,0 +1,173 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using Xunit;
+
+namespace System.DirectoryServices.Protocols.Tests
+{
+    public class DirectoryAttributeModificationCollectionTests
```

Comment:
Instead of writing lots of custom collection tests, can we instead easily reuse the collection tests in common that are meant to help test any collection type?